desc/objectives/_free_boundary.py

```diff
@@ -572,8 +572,6 @@ def __init__(
         field_grid=None,
         name="Quadratic flux",
     ):
-        if target is None and bounds is None:
-            target = 0
         self._src_grid = src_grid
         self._eval_grid = eval_grid
         self._s = s
```

Comment:
Adding a flag like `eq_fixed=False` to this would be good, for cases where we are only optimizing over the external field (coilset i.e. stage 2 optimization) and not the equilibrium, in this case we can precompute all equilibrium quantities once and store them, and also reduces the DOFs to just the external field.

 See e.g. [PlasmaVesselDistance](https://github.com/PlasmaControl/DESC/blob/5d8c1384824133596f09931592fac58504bd3aa3/desc/objectives/_geometry.py#L534) for an example